### ISSUE TITLE
Unable to refresh the pad xy element

### ISSUE THREAD

**Issue by Kallys** [aterrien/jQuery-Kontrol#5]
Hi,

I want to update dynamically parameters of my widgets.
For a dial, I update the value attribute of the html dial element, and then call $('.dial').trigger('change'). It's works pretty fine (but I don't know if it's the better way to do that).
But when I try to do the same with my pad element, I update both x and y hidden inputs of the .pad fieldset, $('.pad).trigger('change') does not refresh the widget (the position of the square isn't updated).

Is that a bug or is there another better way to do this ?

Thanks !
PS: Sorry if I'm not very clear, english is not my mother tongue.

**Comment by aterrien** [comment]
It was not implemented on XY pad.

'change' must be triggered on each input : $('input[name=x]').val(27).trigger('change');

Thank you for your feedback.

**Comment by Kallys** [comment]
Ok thanks, such a pity for XY pad.
Triggering every inputs works fine with bars.
Your lib is cool, I hope a way to update a XY pad will be soon released !

**Comment by aterrien** [comment]
"It was not implemented on XY pad." means it's ok now, you can update XY pad since this issue is closed :)

**Comment by Kallys** [comment]
Ho sorry !
I tried your fix but there is still few problems with XY pad :

1) First, I had to add the var keyword on line 693 for making your code working :
for(i in this.v) { => for(var i in this.v) {

2) Then, refreshing coordinate works fine, but it seems that there is a problem with coordinates. For exemple, when I tried to put the square on (0,0), it appears in (max, 0). When I put on (10, 10), it appears on (max+10, 10). Sounds like if x abscissa had it's value added to max when I send a change event. I hope it's not my code and i'm not wasting your time.

Thanks!
